<?xml version='1.0'?>
<ns2:test-suite xmlns:ns2='urn:model.allure.qatools.yandex.ru' start='1718343697965' stop='1718343720018'>
    <name>Verify user is able to do the payment</name>
    <title>Verify user is able to do the payment</title>
    <test-cases>
        <test-case start='1718343697965' status='failed' stop='1718343720017'>
            <name>Proceed to pay and Confirm Payment Success Message</name>
            <title>Proceed to pay and Confirm Payment Success Message</title>
            <labels>
                <label name='language' value='javascript'/>
                <label name='framework' value='wdio'/>
                <label name='thread' value='0-0'/>
            </labels>
            <parameters>
                <parameter kind='argument' name='device' value='RZ8T50ZVKPE-14'/>
            </parameters>
            <steps/>
            <attachments>
                <attachment title='Screenshot' source='d43575f6-9659-4aea-a549-aa34bb7bfcd7-attachment.png' type='image/png' size='105108'/>
            </attachments>
            <failure>
                <message>Success Message is same: expected '' to equal 'Payment Successful'      
                      actual expected
                      
                      'Payment Successful'
                      </message>
                <stack-trace>AssertionError: Success Message is same: expected '' to equal 'Payment Successful'
                    at CardDetailsScreen.verifySuccessMessage (cardDetails.screen.js:70:55)
                    at processTicksAndRejections (node:internal/process/task_queues:96:5)
                    at async Context.&lt;anonymous> (paymentScenario.spec.js:19:9)</stack-trace>
            </failure>
        </test-case>
    </test-cases>
</ns2:test-suite>